inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/CreateSortConfigListenerV2.java

```diff
@@ -194,7 +204,7 @@ private List<NodeRelation> createNodeRelationsForStream(List<StreamSource> sourc
     }
 
     private void upsertDataFlow(InlongGroupInfo groupInfo, InlongGroupExtInfo extInfo) {
```

Comment:
Maybe function name `upsertDataFlow` need to change. Because it has been expanded to upsert other field.